client/mmadness/src/central/start-round-question/start-round-question.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './../central.css';
+
+export default class StartRoundQuestion extends Component {
+  render() {
+      return (
+        <div className="centralApp">
+          <h1>{this.props.question}</h1>
+          <h4>Use your phone to select a GIF to fit the line above</h4>
+          <br/>
```

Comment:
Don't use `br` for spacing, wrap the "Users in room" in a `span` or a `p` and style it.